### DISCUSSION TITLE
`blog-post.html` template is missing from netlify-template repo

### DISCUSSION THREAD

**Discussion by andrey-jef** [ObsidianPublisher/obsidian-github-publisher#252] [Q&A — Template] [answered]
Hi Mara,

In your [netlify template](https://github.com/ObsidianPublisher/template-netlify-vercel/tree/main), i cannot find the `blog.html` and `blog-post.html`, created by the built-in blog plugin of mkdocs-material.

Do I need to create a `template` folder in my own repo with the `blog.html` and `blog-post.html`? Or u can add them in your template repo, and I can sync to update later?

Fyi, I don't setup mkdocs-material locally, but only clone your netlify template and build with netlify.

**Comment by Mara-Li** [comment, marked as answer]
These are created during the build of material. You can't access it directly, unless you create them to edit it.

Please refere to the documentation of material to do so

**Reply by andrey-jef** [reply]
Thank you for clarification.

My dream of using built-in blog plugin without having to build locally is done. I have to setup on local machine and push to repo then.

**Reply by andrey-jef** [reply]
Hi,

I have just found that I can simply solve the issue with built-in blog plugin by simply deleting the file "blog.html" from "override" folder in my template repo. So i can keep using mkdocs with github-publisher, without having to build locally

Somehow i saw that Mara already updated the sync repo to the usage of "article.html" template, but did not remove the old "blog.html", which is the reason of conflict with the "blog.html" from "template" folder when deploying to Netlify. Or Mara might had guided users to delete the file manually, but somehow I missed that

**Reply by Mara-Li** [reply]
Thanks for the reminder, i will delete the file ! I totally forgot to delete it, yes. The problem to works fully alone...
